lib/resources/example_json/r4_examples_encounter.rb

```diff
@@ -0,0 +1,318 @@
+          },
+          "text": {
+            "status": "generated",
+            "div": "&lt;div>&lt;p>&lt;b>Encounter&lt;/b>&lt;/p>&lt;p>&lt;b>Location&lt;/b>: Baseline East Medical Center, BE Hospital, 1N, 0152, A&lt;/p>&lt;p>&lt;b>Type&lt;/b>: Inpatient&lt;/p>&lt;p>&lt;b>Class&lt;/b>: inpatient encounter&lt;/p>&lt;p>&lt;b>Status&lt;/b>: In Progress&lt;/p>&lt;p>&lt;b>Period Start Date&lt;/b>: Jun 22, 2016  9:05 P.M. UTC&lt;/p>&lt;p>&lt;b>Attending Physician&lt;/b>: Who, Doctor&lt;/p>&lt;/div>"
+          },
+          "identifier": [{
+            "use": "usual",
+            "type": {
+              "coding": [{
+                "system": "https://fhir.cerner.com/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/codeSet/319",
+                "code": "1077",
+                "display": "FIN NBR",
+                "userSelected": true
+              }],
+              "text": "FIN NBR"
+            },
+            "system": "urn:oid:3.3.3.3.3.3",
+            "value": "20003412",
+            "period": {
+              "start": "2016-06-22T21:06:58.000Z"
+            }
+          }],
+          "status": "in-progress",
+          "class": {
+            "system": "http://terminology.hl7.org/CodeSystem/v3-ActCode",
+            "code": "IMP",
+            "display": "inpatient encounter",
+            "userSelected": false
+          },
+          "type": [{
+            "coding": [{
+              "system": "https://fhir.cerner.com/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/codeSet/71",
+              "code": "309308",
+              "display": "Inpatient",
+              "userSelected": true
+            }],
+            "text": "Inpatient"
+          }],
+          "subject": {
+            "reference": "Patient/4342010",
+            "display": "SMART, JOE"
+          },
+          "participant": [{
+            "type": [{
+              "coding": [{
+                "system": "https://fhir.cerner.com/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/codeSet/333",
+                "code": "1116",
+                "display": "Admitting Physician",
+                "userSelected": true
+              }],
+              "text": "Admitting Physician"
+            }],
+            "period": {
+              "start": "2016-11-09T18:07:16.000Z"
+            },
+            "individual": {
+              "reference": "Practitioner/1",
+              "display": "Who, Doctor"
+            }
+          }, {
+            "type": [{
+              "coding": [{
+                "system": "https://fhir.cerner.com/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/codeSet/333",
+                "code": "1119",
+                "display": "Attending Physician",
+                "userSelected": true
+              }],
+              "text": "Attending Physician"
+            }],
+            "period": {
+              "start": "2016-06-22T21:06:58.000Z"
+            },
+            "individual": {
+              "reference": "Practitioner/1912007",
+              "display": "Song, River"
```

Comment:
Nitpick: For participants lets use different ids/displays. Otherwise it could be slightly confusing.

Reply:
The id here should be fine, but picking a different character would be alright since we use River Song in various other examples

Reply:
I just mean in terms of a list of participants having multiple repeated participants. That wouldn't be possible in FHIRPLAY.

Reply:
Ah never mind, it looks like I was wrong. Apparently a single participant can appear multiple times with different types.

Reply:
went ahead and changed: https://github.com/cerner/fhir.cerner.com/pull/238/commits/761f781e1eb58169b9b02d2530719b468b68a24d#diff-8915968a455f59526d7e4635365d9792L98